### ISSUE TITLE
new release? (Extending Search)

### ISSUE THREAD

**Issue by loekwetzels** [TitasGailius/nova-search-relations#29]
Hi Titas, the readme.md shows an example for [Extending Search](https://github.com/TitasGailius/nova-search-relations#extending-search). This feature has not been included in the latest release (1.0.6). Can you give us an update / ETA on the new release?

I have a use case for this feature:
We have a `User` model that as a `settings` relationship. These `settings` have a `key` and a `value`. We want to make these `values` searchable.

So normally you would do something like this on the `Nova/User.php` resource:
```php
public static $searchRelations = [
    'settings' => ['value'],
];
```

We want to apply additional limitations to this search relationship though, because not all `settings.keys` need to searchable. I think we can solve this using the custom search logic you're offering in the (unreleased) version of this package.

**Comment by BobbyBorisov** [comment]
@TitasGailius could you release the new changes like @loekwetzels mentioned? Thank you in advance! Cheers

**Comment by TitasGailius** [comment]
Hey,
I released it as `2.0.0` to avoid any breaking changes just in case.

**Comment by loekwetzels** [comment]
Thank you!
